spec/models/comment_spec.rb

```diff
@@ -1,29 +1,29 @@
-require 'test_helper'
+require 'rails_helper' 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.